charts/karpenter/templates/manifests.yaml

```diff
@@ -4,15 +4,15 @@ metadata:
   creationTimestamp: null
   name: karpenter-mutating-webhook-configuration
   annotations:
-    cert-manager.io/inject-ca-from: karpenter/karpenter-serving-cert
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/karpenter-serving-cert
```

Comment:
This YAML is generated by ./hack/codegen.sh. Can you inject those changes there?

Reply:
Thanks, missed that. Updated codegen.sh, ran it and commited changes.